### PR TITLE
cpusupport.sh: clarify hardcoded flags, nitpick

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
Split from #445.  There's more changes to `cpusupport.sh` now, and it's not technically part of apisupport so I think it looks better as a separate branch / PR.

File: cpusupport/Build/cpusupport.sh [hunk 1]
```diff
@@ -9,17 +9,21 @@ fi
...
 SRCDIR=$(command -p dirname "$0")
 
+CFLAGS_HARDCODED="-D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700"
+
 feature() {
 	ARCH=$1
 	FEATURE=$2
 	shift 2;
-	if ! [ -f ${SRCDIR}/cpusupport-$ARCH-$FEATURE.c ]; then
+	if ! [ -f "${SRCDIR}"/cpusupport-"$ARCH"-"$FEATURE".c ]; then
```
Comment (cperciva) [inline cpusupport/Build/cpusupport.sh:18]: Why not just quote the entire path?

Comment (gperciva) [reply cpusupport/Build/cpusupport.sh]: > Why not just quote the entire path?

I had a vague memory of somebody saying that it was better style to only quote variables.

In any case, when I looked at this file again, I noticed that this mess was defined twice.  I've moved it to a separate variable, which makes the function easier to read.

**Comment by gperciva** [comment]
Revised, including adding a few more comments and the above-mentioned variable.

**Pull request merged**